documentation/manual/filters.md

```diff
@@ -24,34 +24,22 @@ The OSHDB is able to cope well even with complex polygons that have many vertice
 
-_ohsome_ filter
----------------
+Simple examples of filters are `type:node and natural=tree` to select trees, or `geometry:polygon and building=*` to filter for buildings. More examples and can be found on the [dedicated filter documentation page](https://github.com/GIScience/oshdb/blob/0.7.0/oshdb-filter/README.md#examples).
 
-An easy way to provide complex [`filter`s](https://docs.ohsome.org/java/oshdb/0.6.4/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#filter(java.lang.String)) is through the functionality of [ohsome filters](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/libs/ohsome-filter#readme), which allow one to define osm data filter in a human-readable syntax. With these one can combine several tag-, type- and geometry-filters with arbitrary boolean operators.
+By using the methods [`Filter.byOSMEntity`](https://docs.ohsome.org/java/oshdb/0.7.0/aggregated/org/heigit/ohsome/oshdb/filter/Filter.html#byOSMEntity(org.heigit.ohsome.oshdb.util.function.OSMEntityFilter)) and [`Filter.byOSHEntity`](https://docs.ohsome.org/java/oshdb/0.7.0/aggregated/org/heigit/ohsome/oshdb/filter/Filter.html#byOSMEntity(org.heigit.ohsome.oshdb.util.function.OSHEntityFilter)) one can define arbitrary callback functions to filter OSM or OSH entities, respectively.
 
 _lambda_ filter
 ---------------
 
-It is possible to define [`filter` functions](https://docs.ohsome.org/java/oshdb/0.6.4/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#filter(org.heigit.bigspatialdata.oshdb.api.generic.function.SerializablePredicate)) that can sort out values after they already have been transformed in a [map](map-reduce.md#map) step.
+It is possible to define [`filter` functions](https://docs.ohsome.org/java/oshdb/0.7.0/aggregated/org/heigit/ohsome/oshdb/api/mapreducer/MapReducer.html#filter(org.heigit.ohsome.oshdb.api.generic.function.SerializablePredicate)) that can sort out values after they already have been transformed in a [map](map-reduce.md#map) step.
 
-Note that it is usually best to use the less flexible, but more performant OSM data filters described above wherever possible, as they can reduce the amount of data to be iterated over right from the start of the query. This is because while the basic filters are applied at the beginning of each query on the full OSM history data directly, the filter lambda functions are only executed after the data has already been computed and transformed.
+Note that it is usually best to use the _ohsome_ filters described above wherever possible, as they can reduce the amount of data to be iterated over right from the start of the query. Lambda filter functions are only executed after the OSM data has already been computed and transformed.
```

Comment:
Didn't we call them _OSHDB_ filters now, since we moved them into the OSHDB repo?
```suggestion
Note that it is usually best to use the _OSHDB_ filters described above wherever possible, as they can reduce the amount of data to be iterated over right from the start of the query. Lambda filter functions are only executed after the OSM data has already been computed and transformed.
```